src/tests/system/tests/test_sssctl.py

```diff
@@ -545,3 +545,292 @@ def test_sssctl__analyze_with_logdir_and_non_root_privileges(client: Client, pro
+        2. Run sssctl user-show for non-existent user
+    :expectedresults:
+        1. User information is displayed correctly
+        2. Command fails for non-existent user
+    :customerscenario: True
+    """
+    u1 = provider.user("user1").add()
+    u2 = provider.user("CamelCaseUser1").add()
+    client.sssd.domain["use_fully_qualified_names"] = str(use_fqdn)
+    client.sssd.domain["case_sensitive"] = str(case_sensitive)
+    client.sssd.start()
+
+    domain_suffix = "@test" if use_fqdn else ""
+    user1_name = f"{u1.name}{domain_suffix}"
+    user2_name = f"{u2.name}{domain_suffix}"
+
+    # Populate cache
+    client.tools.getent.passwd(user1_name)
+    client.tools.getent.passwd(user2_name)
+
+    result = client.sssctl.user_show(user1_name)
+    assert result.rc == 0
+    assert f"Name: {user1_name}" in result.stdout
+
+    result = client.sssctl.user_show(user2_name)
+    assert result.rc == 0
+    expected_name = user2_name if case_sensitive else user2_name.lower()
+    assert f"Name: {expected_name}" in result.stdout
+
+    # Test case sensitivity for lookups
+    lc_user_name = f"{u2.name.lower()}{domain_suffix}"
+    result = client.sssctl.user_show(lc_user_name)
+    assert result.rc == 0
+    if case_sensitive:
+        assert " is not present in cache" in result.stdout
+    else:
+        assert f"Name: {lc_user_name}" in result.stdout
+
+
+@pytest.mark.parametrize(
+    "use_fqdn,case_sensitive",
+    [
+        (False, True),
+        (True, True),
+        (True, False),
+    ],
+    ids=["shortnames", "fqdn", "fqdn_case_insensitive"],
+)
+@pytest.mark.tools
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__group_show(client: Client, provider: GenericProvider, use_fqdn: bool, case_sensitive: bool):
+    """
+    :title: sssctl group-show displays group information
+    :setup:
+        1. Add groups and users to provider
+        2. Configure SSSD with fqdn and case_sensitive options
+        3. Start SSSD
+        4. Populate cache
+    :steps:
+        1. Run sssctl group-show for groups
+        2. Run sssctl group-show for non-existent group
+    :expectedresults:
+        1. Group information is displayed correctly
+        2. Command fails for non-existent group
+    :customerscenario: True
+    """
+    user = provider.user("user1").add()
+    camel_user = provider.user("CamelCaseUser1").add()
+    provider.group("group1").add().add_member(user)
+    provider.group("CamelCaseGroup1").add().add_member(camel_user)
+    client.sssd.domain["use_fully_qualified_names"] = str(use_fqdn)
+    client.sssd.domain["case_sensitive"] = str(case_sensitive)
+    client.sssd.start()
+
+    domain_suffix = "@test" if use_fqdn else ""
+    group1_name = f"group1{domain_suffix}"
+    cc_group_name = f"CamelCaseGroup1{domain_suffix}"
+
+    # Populate cache
+    client.tools.getent.group(group1_name)
+    client.tools.getent.group(cc_group_name)
+
+    result = client.host.conn.exec(["sssctl", "group-show", group1_name])
```

Comment:
I like to extend the framework so group-show can be used like ```client.sssctl.user_show(lc_user_name)```. Do you want to do that?